lib/SteamCondenser.php

```diff
@@ -23,5 +24,7 @@
  * @return Logger The requested Monolog logger
  */
 function getLogger($name) {
-    return new Logger($name);
+    $logger = new Logger($name);
+    $logger->pushHandler(new StreamHandler("php://stdout"));
+    return $logger;
```

Comment:
Not adding a handler to the Monolog logger was a deliberate decision. That way nobody needs to know how to disable logging. Which is requested more often than vice-versa.

Reply:
I'm a tad bit confused on this, then.

Unit tests with the master branch printed log lines, and before I pushed an STDOUT handler, the tests didn't.

How is the log handling being accomplished?

Reply:
Also, is @mavprolucky you or something? I'm very confused about these comments being left on this PR.

Reply:
Maybe this is new behavior in Monolog 2?

Mavprolucky is not me, just reported as spam.

Reply:
Gotcha, thanks for clarifying, disregard my email then :)

So, what is the desired behavior here? I'm not super sure how to proceed with this.

Reply:
The desired behavior would be „let the user decide and don’t spam whatever logging method by default“.

Reply:
Well, that's doable, do we want to just expose the logger via a getter and setter, and a user could get the logger and push whatever handlers they like to it if they want output?

Reply:
About 2.0 behavior: https://github.com/Seldaek/monolog/commit/ad37b7b2d11f300cbace9f5e84f855d329519e28